.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  bintray-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Run with Gradle
+      env:
+        BINTRAY_USER: ${{ secrets.BintrayUser }}
```

Comment:
The GitHub Actions docs are confusing when it comes to secrets because they mix _PascalCase_ with _SCREAMING_CAPS_ naming conventions. However, they use _SCREAMING_CAPS_ for their secrets (e.g., `secrets.GITHUB_TOKEN`), and their screenshots show _SCREAMING_CAPS_ being used (as are secrets named even outside GitHub Actions such as Unix environment variables). So, change all environment variables to use _SCREAMING_CAPS_.

Reply:
According to the [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#naming-your-secrets) you cannot use the `GITHUB_` prefix though, so I would need to rename `GITHUB_TOKEN` to something else.